README.rst

```diff
@@ -0,0 +1,16 @@
+lmfit
+*****
+
+This is a package providing a nice interface to nonlinear least-
+squares minimization.
+
+Home:
+https://github.com/lmfit/lmfit-py/
+http://lmfit.github.io/lmfit-py/
+
+PyPI site:
+https://pypi.python.org/pypi/lmfit/
+
+Download URL:
+https://pypi.python.org/packages/source/l/lmfit/lmfit-0.9.3.tar.gz
```

Comment:
Please make it clear that these links refer to the `lmfit` distribution itself and not this EUPS implementation.